src/Maestro/Client/src/MaestroApiOptions.cs

```diff
@@ -2,16 +2,33 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.IO;
+using System.Collections.Generic;
 using Azure.Core;
 using Azure.Core.Pipeline;
+using Microsoft.DotNet.Maestro.Common;
 
 
 namespace Microsoft.DotNet.Maestro.Client
 {
     public partial class MaestroApiOptions
     {
-        public static readonly string AUTH_CACHE = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".darc");
+        public const string ProductionBuildAssetRegistryBaseUri = "https://maestro.dot.net/";
+
+        public const string StagingBuildAssetRegistryBaseUri = "https://maestro.int-dot.net/";
+
+        public const string OldProductionBuildAssetRegistryBaseUri = "https://maestro-prod.westus2.cloudapp.azure.com/";
+
+        public const string OldStagingBuildAssetRegistryBaseUri = "https://maestro-int.westus2.cloudapp.azure.com/";
+
+        private const string APP_USER_SCOPE = "Maestro.User";
+
+        private static readonly Dictionary<string, string> EntraAppIds = new Dictionary<string, string>
+        {
+            [StagingBuildAssetRegistryBaseUri.TrimEnd('/')] = "baf98f1b-374e-487d-af42-aa33807f11e4",
+            [OldStagingBuildAssetRegistryBaseUri.TrimEnd('/')] = "baf98f1b-374e-487d-af42-aa33807f11e4",
+            [ProductionBuildAssetRegistryBaseUri.TrimEnd('/')] = "54c17f3d-7325-4eca-9db7-f090bfc765a8",
+            [OldProductionBuildAssetRegistryBaseUri.TrimEnd('/')] = "54c17f3d-7325-4eca-9db7-f090bfc765a8",
```

Comment:
nit: we could also pull these 2 guids in named constants - `MaestroStagingAppId` and `MaestroProductionAppId`

Reply:
Sure, will do